### DISCUSSION TITLE
Local Difficulty: ?? in Multiplayer

### DISCUSSION THREAD

**Discussion by Gho0sti** [FabricMC/fabric-api#3667] [Player support]
### Description

Uhm.. hi!
I dont know how to explain but ive been playing on a Multiplayer Server with Fabric 1.20.4 and in the F3 Menu, the Local Difficulty only shows '??'. No daycount and no Difficulty of the Chunk. This has been happening on all Multiplayer Worlds ive created in Fabric 1.20.4. I've asked the company that hosts these Servers and they have no clue either, but they tested it and had the same issue. Can anyone help me?
![2024-03-24_19 07 11](https://github.com/FabricMC/fabric/assets/164874904/66f5a052-ee59-423d-8c26-59d3103857a0)

**Comment by apple502j** [comment]
Works as intended.

**Reply by Gho0sti** [reply]
What do you mean by that? Is there a way to make it not hide it?

**Reply by Linguardium** [reply]
its literally a vanilla change to the vanilla server and client. fabric does not modify this behaviour. the game is display information the way the vanilla game displays information

**Reply by Gho0sti** [reply]
English isnt my first language, so youre saying, that the server hosts itself thought, it would be a great idea to hide this information?

**Reply by Linguardium** [reply]
Mojang. its a Mojang change to the Unmodified minecraft game

**Comment by Linguardium** [comment]
https://bugs.mojang.com/browse/MC-230732 - bug that was fixed by changing it to ??
https://bugs.mojang.com/browse/MC-265425 - request to change it

this is a vanilla thing

**Reply by Gho0sti** [reply]
Ive already red these posts, but they dont tell you how to solve it. I just want the daycount and the difficulty to be shown again :(

**Reply by Linguardium** [reply]
it isnt something to solve. the values it showed were incorrect as they were not received from the server. instead of hiding the incorrect value lines, mojang chose to just show ??.  

They never showed _correct_ values.

**Reply by Gho0sti** [reply]
in singleplayer, it works but not on the server.. 
well, is there at least a way to get the daycount back?
